lib/tapioca/generator.rb

```diff
@@ -420,7 +420,7 @@ def compile_gem_rbi(gem)
       compiler = Compilers::SymbolTableCompiler.new
       gem_name = set_color(gem.name, :yellow, :bold)
       say("Compiling #{gem_name}, this may take a few seconds... ")
-
+      return if compiler.compile(gem) == "\n"
```

Comment:
This means the gem will be compiled twice in the case where we don't return early. We can use the return value of the `compile(gem)` call instead of the second call down below.

Reply:
You are absolutely correct @KaanOzkan.  I am going to create a local variable to keep this value.